devenv/utils/withdraw.sh

```diff
@@ -36,4 +36,4 @@ fi
 
 tx=$(echo -n $json | jq -r .hex)
 
-sbtc broadcast localhost:60401 $tx | jq -r .
```

Comment:
Why did this work before?

Reply:
We all had installed sbtc binary locally..